<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivityWizardRepetitiveDatesPage</class>
 <widget class="QWizardPage" name="ActivityWizardRepetitiveDatesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>444</height>
   </rect>
  </property>
  <property name="title">
   <string>Activity Dates, Days, Time and Duration</string>
  </property>
  <property name="subTitle">
   <string>Pick activity days or dates and set its time and duration.</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="pickerLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="paso::widget::ItemsPicker" name="itemsPicker" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Start date</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateEdit" name="startDateEdit">
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Finish date</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="finishDateEdit">
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QTimeEdit" name="durationTimeEdit">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="displayFormat">
      <string notr="true">H:mm</string>
     </property>
     <property name="time">
      <time>
       <hour>1</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Start time</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTimeEdit" name="startTimeEdit">
     <property name="displayFormat">
      <string notr="true">H:mm</string>
     </property>
     <property name="time">
      <time>
       <hour>8</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>paso::widget::ItemsPicker</class>
   <extends>QWidget</extends>
   <header>itemspicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
